src/components/MenteeDashboard/userProfile/EditProfileForm.js

```diff
@@ -59,6 +61,12 @@ const EditProfileForm = ({
   }, [error, loading]);
 
   const [country, setCountry] = useState();
+  const [technologies, setTechnologies] = useState([
```

Comment:
Codacy found an issue: ['setTechnologies' is assigned a value but never used.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=6064832)